### PR TITLE
Make desktop version

### PR DESCRIPTION
**Pull request by AdrianCSM**
I made the desktop version using media queries.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -608,3 +615,186 @@ textarea {
...
+    align-items: center;
+    justify-content: center;
+  }
+
+  .descriptionReverse {
+    order: -1;
+    padding: 24px;
+    margin-top: 32px;
+  }
+}
```
Comment (Whoistolu) [inline style.css:800]: - [ ] [REQUIRED] Great desktop design 👍🏽 . Looks almost like the one provided in the template, however, your mobile versions have some issues you need to put in place. The cards are no more at the center of the page. Please fix this to get the project approved.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 |

**Review by codepantherr — CHANGES_REQUESTED** [review]
Hi @AdrianCSM, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## To highlight ⚡ 
- Descriptive readme ✔️ 
- Beautiful UI ✔️ 
- Clean code ✔️ 


## Suggested changes ♻️ 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @AdrianCSM,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [REQUIRED] Kindly update the README file, you currently have a README that has the "Hello Microverse project" as its coming link, please change it to the link of this project. Also, the README does not have a description, MIT license, your social links. Take a look at the [Microverse Readme Template](https://github.com/microverseinc/readme-template) and personalize it.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -608,3 +615,186 @@ textarea {
...
+
+@media screen and (min-width: 992px) {
+  .dropdown {
+    display: none;
+  }
+
+  .mainContainer {
+    /* width: auto; */
+    background-image: url(/images/desktop/desktopBg.svg);
```
[the remark this answers is not in the document]
Comment (AdrianCSM) [reply style.css:630]: I don't know why you can not see it, but it works for me and it doesn't show the same background for mobile. @codepantherr maybe try another browser or other resolutions. As you can see, the other reviewer said nothing about the background so the problem is not on my end.

File: style.css (hunk 1 above)
Comment (AdrianCSM) [reply style.css]: Thank you very much for pointing this out! I didn't even realize that I forgot to add the padding back after I got rid of the bottom scroll bar!

Comment (Whoistolu) [reply style.css]: You are welcome!

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @AdrianCSM , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Decriptive readme file.
:heavy_check_mark: Well implemented Figma design.

### Optional suggestions :diamond_shape_with_a_dot_inside: 

- [ ] For readability, please add two spaces of indentation in your `HTML` file. Do not use the tab key. Please fix it . [For the HTML & CSS best practices, ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

- [ ] Your contact form in the footer does not match with the Figma design for the desktop version :desktop_computer: . Kindly work on that. :writing_hand: 

Your Implementation:
f1
Figma design:
f2




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by prolajumokeoni** [comment]
- Hi @AdrianCSM, You currently have a horizontal scroll on your website, kindly fix this, to make your website better, Weldone completing this project 💪🏿 

https://user-images.githubusercontent.com/69638013/154919459-245b0cac-6ba3-4150-ba4d-aeae30863ebe.mp4

**Review by prolajumokeoni — COMMENTED** [review]
Hi @AdrianCSM 

Your project is deployed!   Congratulations! 🎉 


all-done-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AdrianCSM** [comment]
@prolajumokeoni I tried to remove the scroll bar like 8 times now. I make changes, it works, and then it appears out of nowhere. Thanks for pointing this out, I will try to remove it again later.

**Comment by prolajumokeoni** [comment]
> @prolajumokeoni I tried to remove the scroll bar like 8 times now. I make changes, it works, and then it appears out of nowhere. Thanks for pointing this out, I will try to remove it again later.

- Check your margins and widths 👍🏿

**Pull request merged**